.travis.yml

```diff
@@ -5,6 +5,7 @@ env:
   - NODE_VERSION="v0.12"
   - NODE_VERSION="iojs-v1.0"
   - NODE_VERSION="iojs-v2.0"
+  - NODE_VERSION="iojs-v3.0"
```

Comment:
This should just be `NODE_VERSION="iojs-v3"`, and all the other `iojs` lines should drop the `.0` as well.